### PR TITLE
Adds Sourcify post to blog 2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Hum…

### PR DESCRIPTION
**Pull request by franzihei**
…ans.md

### CODE REVIEW

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+published: true
+title: 'Sourcify: Towards Safer Contract Interaction for Humans'
+date: '2020-05-29'
+author: Edi Sinovčić, Franziska Heintel
```
Comment (edisinovcic) [inline _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:6]: You can also add Rob Stupay as he helped a lot also.

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+You might already know Sourcify under the name “Source-Verify”/ ”Source-Verifier” - don’t despair, we just decided to give 
+it a nicer name!
+
+## YOLO Signing - One of Today’s Everyday Problems in Ethereum UX
+Think about the following: How many developers of the dapps you are using do you know personally?
+
+Let’s imagine you want to interact with a dapp using MetaMask and it shows you this dialogue: 
+
+![](https://i.imgur.com/CjzXm9p.png)
```
Comment (axic) [inline _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:26]: Should include the image in the repository.

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+## How does Sourcify work under the hood?
+
+Sourcify is a project on the Ethereum blockchain that:
+* Listens for the deployment of smart contracts.
+* Checks to see if their metadata and source code is published on IPFS or Swarm.
+* Recompiles the code.
+* Checks that it is the same as the deployed code.
+* Permanently stores the source code, ABI and other metadata on IPFS.
+
+![](https://i.imgur.com/FewKPRF.png)
```
Comment (axic) [inline _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:69]: Same here.

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+Ethereum Natural Language Specification Format which describes functions, return values etc. in a friendly way.
+1. Publishing the dapp’s source code and its metadata to a decentralized network.
+
+## Meet Sourcify, a decentralized automated contract verifying service
+
+Sourcify tries to make the process of source verification (step 2 in the list above) easy for you. There are different ways 
+how you can make use of the tool. 
+
+You can either...
+* Use the interface hosted on verification.komputing.org
```
Comment (axic) [inline _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:54]:
```suggestion
* Use the interface hosted on [verification.komputing.org](https://verification.komputing.org/)
```

Should also really consider having a sourcify domain :)

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+author: Edi Sinovčić, Franziska Heintel
+category: Announcements
+---
+
+**tl;dr**: Building sensible blockchain applications for humans is **hard**. You can enhance the user experience of 
+your dapp today by leveraging the power of open source. Increase awareness and give more transparency on what users 
+are *actually* doing when interacting with your code on the blockchain, i.e. when signing a transaction, by publishing 
+the source code to this decentralized repository and using metadata files, which translate “random” hex strings into 
+human-readable language. Sourcify is a tool to help you do exactly that. If you want to upload your source code, either 
+visit verification.komputing.org, the [Sourcify Github repo](https://github.com/ethereum/sourcify) or use the Sourcify
```
Comment (axic) [inline _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:15]:
```suggestion
visit [verification.komputing.org](https://verification.komputing.org/), the [Sourcify Github repo](https://github.com/ethereum/sourcify) or use the Sourcify 
```

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md (hunk 4 above)
Comment (franzihei) [reply _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:54]: yeah totally...

**Pull request merged**